load-agent/agent.ts

```diff
@@ -74,49 +90,72 @@ const initializeAgent = async (withMediation, port, agentConfig = null) => {
+          connectOnStartup: true,
+        }
+      ]
+    }),
+    askar: new AskarModule({
+      ariesAskar,
+    }),
+    mediator: new MediatorModule({
+      autoAcceptMediationRequests: true,
```

Comment:
Some configurations don't use mediation, so we want to use mediation down below. See "if (withMediation) {"

Reply:
Ok, will push the code with changes

Reply:
Fixed